src/intrinsics.cr

```diff
@@ -1,6 +1,6 @@
 lib Intrinsics
   fun debugtrap = "llvm.debugtrap"
-  {% if flag?(:x86_64) %}
+  {% if flag?(:x86_64) || flag(:aarch64) %}
```

Comment:
I thought of that too. Then I thought that maybe `SizeT` could possibly be the wrong type if the platform's libc is weird. But I think that's such an abstract worry that adding flags is probably the best solution.